src/storage/StorageServer.cpp

```diff
@@ -42,6 +42,8 @@ DEFINE_bool(local_config, false, "meta client will not retrieve latest configura
 DEFINE_int32(storage_port, 44501, "Storage daemon listening port");
 DEFINE_int32(storage_num_worker_threads, 32, "Number of workers");
 DECLARE_bool(local_config);
+DEFINE_bool(add_local_host, true, "Whether add localhost automatically");
```

Comment:
should we tell the user "in standalone version"?

Reply:
Only standalone version supply this config, and only standalone config file contain it.